### PR TITLE
Add a view documentation link to the PlacementRule deprecation banner

### PR DESCRIPTION
**Pull request by mprahl**
This varies from the mock up design to conform to other parts of the console.

image

Relates:
https://issues.redhat.com/browse/ACM-4744

### CODE REVIEW

**Comment by mprahl** [comment]
@dockerymick could you please review this?

File: frontend/src/components/PlacementRuleDeprecationAlert.tsx [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
+import { viewDocumentation, DOC_LINKS } from '../lib/doc-util'
 
 const PlacementRuleDeprecationAlert = () => {
   const { t } = useTranslation()
   return (
     <React.Fragment>
       <Alert variant="info" isInline title={t('Placement rule deprecation')} style={{ marginBottom: '10px' }}>
-        <p>{t('PlacementRule is deprecated and will not receive updates or fixes. Best practice: Use Placement.')}</p>
+        <TextContent>
+          {t('PlacementRule is deprecated and will not receive updates or fixes. Best practice: Use Placement.')}
```
Comment (dockerymick) [inline frontend/src/components/PlacementRuleDeprecationAlert.tsx:14]: @mprahl 

I suggest the following updates:

Change PlacementRule is deprecated and will not receive updates or fixes.

TO 

Placement rule is deprecated and will not receive updates or fixes.
OR
PlacementRule resource is deprecated and will not receive updates or fixes.

Not sure if we can move Best practice to the next line and bold "Best practice:"

**Comment by mprahl** [comment]
@dockerymick could you please provide feedback on the latest iteration?
image

**Comment by mprahl** [comment]
The test failure is unrelated to this PR.

File: frontend/public/locales/en/translation.json [hunk 2]
```diff
@@ -30,6 +30,7 @@
   "< Back to all {{resourceType}} resources": "< Back to all {{resourceType}} resources",
   "<0>Run this command</0> to import your cluster": "<0>Run this command</0> to import your cluster",
   "<0>Run this command</0> to log-in to the existing cluster in your terminal": "<0>Run this command</0> to log-in to the existing cluster in your terminal",
```
Comment (dockerymick) [inline frontend/public/locales/en/translation.json:32]: Remove the hyphen that is in between log and in (`log-in`). Update to `log in

**Review by dockerymick — APPROVED** [review]

**Comment by mprahl** [comment]
> @mprahl There's a failing test currently. Can you please check if it fails consistently?
> 
> Also - where did you get the translations? In general, you only need to update the English file, and the translation team will update our languages.

Thanks! They were a best guess since it was just splitting up the existing text but I removed it from non-English translation files.

I just rebased and will see if the unrelated test failure is fixed.

/unhold

**Comment by mprahl** [comment]
This looks like an issue in this PR. I'll investigate.

**Review by dockerymick — APPROVED** [review]

File: frontend/public/locales/ko/translation.json [hunk 3]
```diff
@@ -1895,7 +1895,6 @@
   "Placement rules define the target clusters where subscriptions are delivered. This is done by cluster name, cluster resource annotation(s), or cluster resource label(s).": "배치 규칙은 서브스크립션이 제공되는 대상 클러스터를 정의합니다. 이는 클러스터 이름, 클러스터 리소스 주석 또는 클러스터 리소스 레이블로 수행됩니다.",
   "Placement rules determine which clusters a resources will be applied.": "배치 규칙은 리소스를 적용할 클러스터를 결정합니다.",
   "PlacementRule": "배치 규칙",
-  "PlacementRule is deprecated and will not receive updates or fixes. Best practice: Use Placement.": "배치 규칙은 더 이상 사용되지 않으며 업데이트 또는 수정 사항을 수신하지 않습니다. 모범 사례: 배치를 사용합니다.",
```
[the remark this answers is not in the document]
Comment (yiraeChristineKim) [reply frontend/public/locales/ko/translation.json:1898]: who translate this though

**Review by yiraeChristineKim — APPROVED** [review]

**Comment by mprahl** [comment]
@KevinFCormier could you please review this again? The last commit explains the CI failure fix.

**Pull request merged**